x-pack/test/security_solution_api_integration/test_suites/entity_analytics/risk_engine/trial_license_complete_tier/asset_criticality_csv_upload.ts

```diff
@@ -0,0 +1,189 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import expect from '@kbn/expect';
+import omit from 'lodash/omit';
+import {
+  assetCriticalityRouteHelpersFactory,
+  cleanAssetCriticality,
+  getAssetCriticalityDoc,
+} from '../../utils';
+import { FtrProviderContext } from '../../../../ftr_provider_context';
+export default ({ getService }: FtrProviderContext) => {
+  describe('@ess @serverless Entity Analytics - Asset Criticality CSV upload', () => {
+    const esClient = getService('es');
+    const supertest = getService('supertest');
+    const assetCriticalityRoutes = assetCriticalityRouteHelpersFactory(supertest);
+    const log = getService('log');
+    const expectAssetCriticalityDocMatching = async (expectedDoc: {
+      id_field: string;
+      id_value: string;
+      criticality_level: string;
+    }) => {
+      const esDoc = await getAssetCriticalityDoc({
+        es: esClient,
+        idField: expectedDoc.id_field,
+        idValue: expectedDoc.id_value,
+      });
+
+      expect(omit(esDoc, '@timestamp')).to.eql(expectedDoc);
+    };
+    before(async () => {
+      await cleanAssetCriticality({ es: esClient, namespace: 'default', log });
+    });
+
+    after(async () => {
+      await cleanAssetCriticality({ es: esClient, namespace: 'default', log });
+    });
+
+    describe('Asset Criticality privileges API', () => {
+      it('should correctly upload a valid csv with one entity', async () => {
+        const validCsv = 'host,host-1,low_impact';
+        const { body } = await assetCriticalityRoutes.uploadCsv(validCsv);
+        expect(body.errors).to.eql([]);
+        expect(body.stats).to.eql({
+          total: 1,
+          created: 1,
+          updated: 0,
+          errors: 0,
+        });
+
+        expectAssetCriticalityDocMatching({
```

Comment:
```suggestion
        await expectAssetCriticalityDocMatching({
```

does this require an `await` for the test framework? Sorry if I'm wrong. It's been a long time since I worked with this.

Reply:
Added, thanks!